benchmark/king2dcompressible/ala/ala.prm

```diff
@@ -0,0 +1,286 @@
+  set Prescribed velocity boundary indicators =
+  set Tangential velocity boundary indicators = left, right, top, bottom
+
+  # The final part of this section describes whether we
+  # want to include adiabatic heating (from a small
+  # compressibility of the medium) or from shear friction,
+  # as well as the rate of internal heating. We do not
+  # want to use any of these options here:
+  set Include adiabatic heating               = false
+  set Include shear heating                   = false
+end
+
+subsection Heating model
+  # shear heating: 1/2 eta*e(u): ... need  -> 1/2 e(u): ..  * Di/Ra
+  # adiabatic heating: alpha T (u . \nabla p)
+  # 
+
+  set List of model names = adiabatic heating, shear heating#, constant heating
+
+  subsection Constant heating
+    set Radiogenic heating rate = 1.25e-12#5e-9
+  end
+
+  subsection Adiabatic heating
+    # A flag indicating whether the adiabatic heating should be simplified
+    # from $\alpha T (\mathbf u \cdot \nabla p)$ 
+    # to $ \alpha \rho T (\mathbf u \cdot \mathbf g) $. alpha=thermal exp
+
+    # have \alpha rho T w
+    # need: \alpha \rho T w Di
+    set Use simplified adiabatic heating = true
+  end
+end
+
+subsection Formulation
+  set Formulation = ALA # equivalent to 'custom' with the settings below
+  #set Mass conservation    = reference density profile
+  #set Temperature equation = reference density profile
+end
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 1
+  end
+end
+
+
+subsection Material model
+  #set Model name = simple compressible # default:
+  set Model name = Material
+  subsection Tan Gurnis model
+    set Reference density             = 1
+
+    # The value of the specific heat $cp$. Units: $J/kg/K$. => H
+    set Reference specific heat       = 1   # ??
+
+    # The reference temperature $T_0$. Units: $K$.
+    set Reference temperature         = 0
+
+    # The value of the constant viscosity. Units: $kg/m/s$.
+    set Ra                     = 1e4
+    set Di = 1.0 # BA: Di=0.0
+    set b=0
+    set c=0
+    set gamma = 1.0
+  end
+
+end
+
+
+# The settings above all pertain to the description of the
+# continuous partial differential equations we want to solve.
+# The following section deals with the discretization of
+# this problem, namely the kind of mesh we want to compute
+# on. We here use a globally refined mesh without
+# adaptive mesh refinement.
+subsection Mesh refinement
+  set Initial global refinement                = 4
+  set Initial adaptive refinement              = 0
+  set Time steps between mesh refinement       = 0
+  set Coarsening fraction                      = 0.0
+  set Refinement fraction                      = 1.0
+  #set Additional refinement times              = 0.1, 0.2, 0.3, 0.5, 1.0, 1.5, 2.0
+  set Additional refinement times              = 0.5, 1.0, 1.5, 2.0
+end
+
+
+# The final part is to specify what ASPECT should do with the
```

Comment:
This an several of the other files repeat a lot of commentary from existing .prm files. I don't know whether that's necessary -- I'll leave this up to you, just wanted to point out the duplication.